src/Oracles/Config/Setting.hs

```diff
@@ -133,6 +148,25 @@ osxHost = anyHostOs ["darwin"]
 windowsHost :: Action Bool
 windowsHost = anyHostOs ["mingw32", "cygwin32"]
 
+-- https://github.com/ghc/ghc/blob/master/mk/config.mk.in#L241
+relocatableBuild :: Action Bool
+relocatableBuild = windowsHost
+
+-- TODO: Prettify this into a builder?
+installDir :: String -> Action ()
```

Comment:
Let's add Haddock to these actions. Also, I suggest to move these definitions to `Util.hs`.